src/coveralls.erl

```diff
@@ -83,8 +84,10 @@ convert_and_send_file(Filename, ServiceJobId, ServiceName) ->
 %%=============================================================================
 %% Internal functions
 
-convert_file(Filename, ServiceJobId, ServiceName, S) ->
-  ok               = import(S, Filename),
+convert_file([L|_]=Filename, ServiceJobId, ServiceName, S) when is_integer(L) ->
+    convert_file([Filename], ServiceJobId, ServiceName, S);
+convert_file([[_|_]|_] = Files, ServiceJobId, ServiceName, S) ->
+  _                = [ok = import(S, Filename) || Filename <- Files],
```

Comment:
use lists:foreach when "looping" over a list for its' side-effects

`lists:foreach(fun(Filename) -> ok = import(S, Filename) end, Files)`